cli/cmd/options.go

```diff
@@ -622,6 +627,16 @@ func validateProxyValues(values *l5dcharts.Values) error {
 	return nil
 }
 
+func validatePolicy(policy string) error {
+	validPolicies := []string{"all-authenticated", "all-unauthenticated", "cluster-authenticated", "cluster-unauthenticated", "deny"}
```

Comment:
tiny nit: since this is a fixed size array, should we use 
```suggestion
	validPolicies := [...]string{"all-authenticated", "all-unauthenticated", "cluster-authenticated", "cluster-unauthenticated", "deny"}
```
here rather than a slice?

i was going to suggest that it should be a constant, but i learned Go does not have `const` arrays

Reply:
I'm going to keep this as a slice mainly because of the error formatting below. `strings.Join` has to take a slice, so if this was changed to an array it would just be converted to a slice later anyways (`strings.Join(validPolicies[:], ", ")`).